<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1600</width>
    <height>1200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="lbE1">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>251</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Leelawadee UI</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/cct/TextE1.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lbE2">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>20</y>
     <width>261</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Leelawadee UI</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/cct/TextE2.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit1">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>70</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit2">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>70</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_start">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>220</y>
     <width>421</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Leelawadee UI</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(238, 238, 238);
</string>
   </property>
   <property name="text">
    <string>Start Simulation</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_res">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>290</y>
     <width>741</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Leelawadee UI</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Elemento 1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbFondo">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>1200</width>
     <height>800</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>1200</width>
     <height>800</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1600</width>
     <height>1200</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="fondo.qrc">:/cct/fondo.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lbResult">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>150</y>
     <width>321</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="Result.qrc">:/cct/Result.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lbTabla">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>430</y>
     <width>981</width>
     <height>351</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="PeriodicT.qrc">:/cct/PeriodicT.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbWrite">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>430</y>
     <width>511</width>
     <height>161</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/cct/TextWES.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>lbFondo</zorder>
  <zorder>lbE1</zorder>
  <zorder>lbE2</zorder>
  <zorder>label_res</zorder>
  <zorder>lbResult</zorder>
  <zorder>pushButton_start</zorder>
  <zorder>lineEdit1</zorder>
  <zorder>lineEdit2</zorder>
  <zorder>lbTabla</zorder>
  <zorder>lbWrite</zorder>
 </widget>
 <resources>
  <include location="TextE1.qrc"/>
  <include location="TextWES.qrc"/>
  <include location="PeriodicT.qrc"/>
  <include location="Result.qrc"/>
  <include location="fondo.qrc"/>
  <include location="TextE2.qrc"/>
  <include location="../../../../../.designer/backup/PeriodicT.qrc"/>
  <include location="../../../../../.designer/backup/Result.qrc"/>
  <include location="../../../../../.designer/backup/fondo.qrc"/>
 </resources>
 <connections/>
</ui>
